### PR TITLE
updated the versions of numpy,scipy & scikit-learn

### PR DESCRIPTION
**Pull request by urstrulykkr**
I had updated according to the latest versions of those specified libraries, according to SemVer.
Requesting to get it review my commit.

### CODE REVIEW

**Comment by urstrulykkr** [comment]
@halfak  I made my patch, hoping to recieve your kind reviews on it :)

File: requirements.txt [hunk 1]
```diff
@@ -12,12 +12,12 @@ mwtypes >= 0.2.0, < 0.3.999
 mwparserfromhell >= 0.5.1, < 0.5.999
 mysqltsv >= 0.0.7, < 0.0.999
 nltk >= 3.0.0, < 3.0.999
-numpy >= 1.14.0, < 1.14.999
+numpy >= 1.16.2, < 2.0.999
```
Comment (halfak) [inline requirements.txt]: I think we'll want to stay within minor version for these ranges.  E.g. `numpy >= 1.16.2, < 1.6.999`.  See https://semver.org/ for the meaning of these numbers.

Comment (urstrulykkr) [reply requirements.txt]: cool. Will apply the review made :)

Comment (urstrulykkr) [reply requirements.txt]: In here `E.g. numpy >= 1.16.2, < 1.6.999` it should be  <1.16.999 right ?

**Comment by urstrulykkr** [comment]
Thanks a lot for your patience and time.
That makes my 3rd patch to Wikimedia :)

Best
Kaushik.

**Pull request merged**